package/react-native-slider.podspec

```diff
@@ -2,6 +2,9 @@ require 'json'
 
 package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
 
+folly_version = '2021.06.28.00-v2'
```

Comment:
Looks like this can be removed https://github.com/software-mansion/react-native-screens/pull/1549

Reply:
Removed ✅